### PR TITLE
Bug 1978421: String updates (typos, etc.)

### PR DESCRIPTION
**Pull request by SanjalKatiyar**

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json [hunk 1]
```diff
@@ -428,7 +428,7 @@
...
   "Local Storage Operator not installed": "Local Storage Operator not installed",
-  "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a storage cluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a storage cluster.<1><0>Install</0></1>",
+  "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the local storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>",
```
Comment (afreen23) [inline frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:431]: The name of an operator is a proper noun so we can use title case.
```suggestion
  "Before we can create a StorageCluster, the Local Storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>": "Before we can create a StorageCluster, the Local Storage operator needs to be installed. When installation is finished come back to OpenShift Container Storage to create a StorageCluster.<1><0>Install</0></1>",
```

**Review by afreen23 — CHANGES_REQUESTED** [review]

**Comment by GowthamShanmugam** [comment]
@SanjalKatiyar can you ask QE to verify this

**Pull request merged**